### PR TITLE
Updated the Project Name

### PR DESCRIPTION
**Pull request by sudheer51**
Modified the File RegPatient.

### CODE REVIEW

File: mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java [hunk 1]
```diff
@@ -12,10 +12,6 @@
 public class US_005_RegisterPatientTests extends TestBaseClass {
```
Comment (archana7pandey) [inline mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java:12]: what is the reason for this change....

File: mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java [hunk 2]
```diff
@@ -12,10 +12,6 @@
 public class US_005_RegisterPatientTests extends TestBaseClass {
 
 	HashMap<String, String> hMapPatientData;
-
-	firstname
-	
-	
 	@Test
```
Comment (archana7pandey) [inline mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java:15]: testing

File: mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java [hunk 3]
```diff
@@ -12,10 +12,6 @@
 public class US_005_RegisterPatientTests extends TestBaseClass {
```
Comment (sudheer51) [inline mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java:13]: What is the change ??

Comment (sudheer51) [inline mmp_iit_1802/src/test/java/org/iit/mmp/mmp_iit/US_005_RegisterPatientTests.java:13]: This is required due to importing the classes

**Review by sudheer51 — COMMENTED** [review]
Approved
